### DISCUSSION TITLE
live traing program graph

### DISCUSSION THREAD

**Discussion by brokkolo** [cagnulein/qdomyos-zwift#1450] [Ideas]
I use the open training programs with my rides.
would it be possible to have a tile with the graph of the whole training program, during the ride, showing where we are as we train?
thinking of the graphs shown at the end of the workout, I imagine a tile during the workout (perhaps double in width) showing the entire resistance of the workout with an overlay of power/cadence/hr (perhaps in different colors and configurable from the tiles settings)

**Comment by cagnulein** [comment]
yes it's in the todo list :) there is already a PR for this https://github.com/cagnulein/qdomyos-zwift/pull/1451 :)
